packages/graphql-language-service/src/interface/getAutocompleteSuggestions.ts

```diff
@@ -1310,14 +1313,12 @@ function unwrapType(state: State): State {
   if (
     state.prevState &&
     state.kind &&
-    (
-      [
-        RuleKinds.NAMED_TYPE,
-        RuleKinds.LIST_TYPE,
-        RuleKinds.TYPE,
-        RuleKinds.NON_NULL_TYPE,
-      ] as RuleKind[]
-    ).includes(state.kind)
+    ([
```

Comment:
It seems the prettier --check wants you to revert this formatting change

Reply:
Interesting, I didn't make this change lol